src/css/base.scss

```diff
@@ -4911,3 +4911,9 @@ textarea::placeholder {
     display: grid;
     grid-template-columns: 50% 50%;
 }
+
+.cd-material {
+    &__container-with-sidebar {
+        grid-template-columns: 234px auto;
+    }
+}
```

Comment:
leave a comment that this should be moved to common whenever the respective component is moved